src/learn/zkcloud/snarkos_dev_guide.md

```diff
@@ -0,0 +1,214 @@
+      --storage_path <STORAGE_PATH>  #Specify the path to a directory containing the ledger
+  -h, --help                         #Print help
+
+```
+
+#### 2.3 Execute Transaction
+```bash
+snarkos developer execute [OPTIONS] --private-key <PRIVATE_KEY> --query <QUERY> <PROGRAM_ID> <FUNCTION> [INPUTS]...
+```
+**[Execute Options]**
+```bash
+ -p, --private-key <PRIVATE_KEY>     #The private key used to generate the execution
+  -q, --query <QUERY>                #The endpoint to query node state from
+      --priority-fee <PRIORITY_FEE>  #The priority fee in microcredits
+  -r, --record <RECORD>              #The record to spend the fee from
+  -b, --broadcast <BROADCAST>        #The endpoint used to broadcast the generated transaction
+  -d, --dry-run                      #Performs a dry-run of transaction generation
+      --store <STORE>                #Store generated deployment transaction to a local file
+      --storage_path <STORAGE_PATH>  #Specify the path to a directory containing the ledger
+  -h, --help                         #Print help
+```
+
+#### 2.4 Scan Records
+```
+snarkos developer scan [OPTIONS] --endpoint <ENDPOINT>
+```
+**[Execute Options]**
+```bash
+-p, --private-key <PRIVATE_KEY>    #An optional private key scan for unspent records
+  -v, --view-key <VIEW_KEY>        #The view key used to scan for records
+      --start <START>              #The block height to start scanning from
+      --end <END>                  #The block height to stop scanning
+      --last <LAST>                #Scan the latest `n` blocks
+      --endpoint <ENDPOINT>        #The endpoint to scan blocks from
+  -h, --help                       #Print help
+```
+
+
+### 3. Local Devnet
+
+#### 3.1 Install `tmux`
+
+To run a local devnet with the script, start by installing `tmux`.
+
+<details><summary>macOS</summary>
+
+To install `tmux` on macOS, you can use the `Homebrew` package manager.
+If you haven't installed `Homebrew` yet, you can find instructions at [their website](https://brew.sh/).
+```bash
+# Once Homebrew is installed, run:
+brew install tmux
+```
+
+</details>
+
+<details><summary>Ubuntu</summary>
+
+On Ubuntu and other Debian-based systems, you can use the `apt` package manager:
+```bash
+sudo apt update
+sudo apt install tmux
+```
+
+</details>
+
+<details><summary>Windows</summary>
+
+There are a couple of ways to use `tmux` on Windows:
+
+### Using Windows Subsystem for Linux (WSL)
+
+1. First, install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install).
+2. Once WSL is set up and you have a Linux distribution installed (e.g., Ubuntu), open your WSL terminal and install `tmux` as you would on a native Linux system:
+```bash
+sudo apt update
+sudo apt install tmux
+```
+
+</details>
+
+#### 3.2 Start a Local Devnet
+
+To start a local devnet, run:
+```bash
+./devnet.sh
+```
+Follow the instructions in the terminal to start the devnet.
+
+#### 3.3 View a Local Devnet
```

Comment:
Suggestion: Add more frequently used tmux command for easier navigation. Like `ctrl+b [` to scroll and `q` to quit scrolling as example.